crates/bevy_pbr/src/render/mesh_vertex_output.wgsl

```diff
@@ -1,13 +1,16 @@
 #define_import_path bevy_pbr::mesh_vertex_output
 
-@location(0) world_position: vec4<f32>,
-@location(1) world_normal: vec3<f32>,
-#ifdef VERTEX_UVS
-@location(2) uv: vec2<f32>,
-#endif
-#ifdef VERTEX_TANGENTS
-@location(3) world_tangent: vec4<f32>,
-#endif
-#ifdef VERTEX_COLORS
-@location(4) color: vec4<f32>,
-#endif
+struct MeshVertexOutput {
+    @builtin(position) clip_position: vec4<f32>,
```

Comment:
Just a note that I think the fragment stage input builtin position isn't in clip space, rather screen space. So texels from 0 to width and 0 to height from the beginning of the texture (should be top-left), and then depth buffer depth in z.

Reply:
hm. i think it is clip position when used as vertex output, but frag coord when used as fragment input? this struct spans both. unfortunately wgsl forces output structs to use no nesting, so i have to include it in the struct (even though in frag shaders it can be read independently of the struct).

Reply:
shall i just call it `position` ?

Reply:
`position` sounds good to me.